config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  resources :boards, only: [:index, :new, :create, :edit, :update] # poprzez podanie resources konfiguruje automatycznie ścieżki, only po aby robić po kolei, bo inaczej stworzy caly komplet
+  resources :boards, only: [:index, :new, :create, :edit, :update, :destroy, :show] # poprzez podanie resources konfiguruje automatycznie ścieżki, only po aby robić po kolei, bo inaczej stworzy caly komplet
```

Comment:
jesli uzywasz juz wszystkich CRUD'owych metod to mozesz wywalic only i resources swtorzy Ci wszystko czego potrzebujesz :)